doc/source/ray-air/getting-started.rst

```diff
@@ -3,37 +3,46 @@
 
+Get started by installing Ray AIR:
 
-Ray AIR focuses on two functional aspects:
+.. code:: bash
 
-* It provides scalability by leveraging Ray’s distributed compute layer for ML workloads.
-* It is designed to interoperate with other systems for storage and metadata needs.
+    pip install -U ray[air]
 
-Ray AIR consists of five key components:
+Why Ray AIR?
+------------
+
+Ray AIR aims to simplify the ecosystem of machine learning frameworks, platforms, and tools. It does this by taking a scalable, single-system approach to ML infrastructure (i.e., leveraging Ray as a unified compute framework):
+
+**1. Seamless Dev to Prod**: AIR reduces friction going from development to production. Traditional orchestration approaches introduce separate systems and operational overheads. With Ray and AIR, the same Python code scales seamlessly from a laptop to a large cluster.
 
-- Data processing (:ref:`Ray Data <datasets>`)
-- Model Training (:ref:`Ray Train <train-docs>`)
-- Reinforcement Learning (:ref:`Ray RLlib <rllib-index>`)
-- Hyperparameter Tuning (:ref:`Ray Tune <tune-main>`)
-- Model Serving (:ref:`Ray Serve <rayserve>`).
+**2. Unified API**: Want to switch between frameworks like XGBoost and PyTorch, or try out a new library like HuggingFace? Thanks to the flexibility of Ray AIR, you can do this by just swapping out a single class, without needing to set up new systems or change other aspects of your workflow.
 
-Users can use these libraries interchangeably to scale different parts of standard ML workflows.
+**3. Open and Evolvable**: Ray core and libraries are fully open-source and can run on any cluster, cloud, or Kubernetes, reducing the costs of platform lock-in. Want to go out of the box? Run any framework you want using AIR's integration APIs, or build advanced use cases directly on Ray core.
 
-To get started, install Ray AIR via ``pip install -U "ray[air]"``
+.. figure:: images/why-air.png
 
+  AIR enables a single-system / single-script approach to scaling ML. Ray's APIs for
+  distributed computation allow ML workloads to be scaled without needing
+  to set up or orchestrate separate systems, reducing the friction of moving from
+  development to production.
+
+AIR is for both ML practitioners and ML engineers. Consider using AIR when you want to:
+ * Scale a single workload.
+ * Scale end-to-end ML applications.
+ * Build a custom ML platform for your organization.
 
 Quick Start
 -----------
 
-Below, we demonstrate how you can use the Ray libraries in a seamless flow
-between distributed frameworks (e.g., XGBoost, Pytorch, and Tensorflow): 
+Below, we demonstrate how AIR enables simple scaling of end-to-end ML workflows
+involving multiple distributed frameworks (here showing XGBoost, Pytorch, and Tensorflow):
```

Comment:
this reads as if each workflow touched multiple frameworks (but it doesn't)

Reply:
May be rephrase is as:
Below, we demonstrate how AIR enables simple scaling of end-to-end ML workflows using one of the multiple distributed supported frameworks (e.g., XGBoost, PyTorch, and Tensorflow):

Reply:
**When to use AIR?**

AIR is for both data scientists and ML engineers. Consider using AIR when you want to:
 * Scale a single workload such as training, tuning, or inference
 * Scale end-to-end ML applications.
 * Build a custom ML platform for your organization.

Reply:
let's use data scientist and ML engineers to be consistent with our slides in the Ray AIR presentation.